config/faf.conf.in

```diff
@@ -30,3 +30,7 @@ CreateComponents = False
 # allowed values: fedora-bugzilla rhel-bugzilla centos-mantisb comment email url
 # or * to allow all attachments
 AcceptAttachments = fedora-bugzilla rhel-bugzilla centos-mantisbt
+
+# allowed values for the Known option
+#BUG_OS_MAJOR_VERSION BUG_OS_MINOR_VERSION EQUAL_UREPORT_EXISTS
+Known = 
```

Comment:
It's not clear what 'Known' does or means in this context. Please mention that it changes uReport matching strategy. Also what is the default value? Looks like it will fail matching any reports if left blank.